### ISSUE TITLE
Use bdk python bindings in place of bitcoind for integration tests

### ISSUE THREAD

**Issue by BitcoinZavior** [LtbLightning/payjoin-ffi#11]
Need arm64 architecture support for bdk-python before bdk can be used for integration tests

**Comment by DanGould** [comment]
Is there a reason we need the complexity of BDK in the bindings project for tests? could we not include a BDK test at the rust-payjoin library level to ensure compatibility and use simple bitcoind tests to test each binding layer?

**Comment by BitcoinZavior** [comment]
> Is there a reason we need the complexity of BDK in the bindings project for tests? could we not include a BDK test at the rust-payjoin library level to ensure compatibility and use simple bitcoind tests to test each binding layer?

Yes thats one option. The reason to use bdk for integration tests was to ensure ffi wrapper works just as well as rust payjoin. Secondly to identify any integration issues when using bdk and thirdly as a guide for bdk users to integrate payjoin.

**Comment by spacebear21** [comment]
Superseded by [Include BDK integration tests in rust-payjoin](https://github.com/payjoin/rust-payjoin/issues/741)
